cluster/gce/config-test.sh

```diff
@@ -316,7 +316,7 @@ if [[ -n "${GCE_GLBC_IMAGE:-}" ]]; then
 fi
 
 if [[ -z "${KUBE_ADMISSION_CONTROL:-}" ]]; then
-  ADMISSION_CONTROL="Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,PodPreset,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority"
+  ADMISSION_CONTROL="MutatingAdmissionWebhook,Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,PodPreset,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority"
```

Comment:
It's on my list to document the correct order for this. I don't know what else I will say, but I'm pretty sure initializers should go before this.

Reply:
Discussed offline, we perhaps want to disallow mutating webhooks to change initializer list to begin with, so I'll keep the current order in this PR.

Reply:
Ah, I wondered where that question came from.  I think having initializers last is reasonable, but I think we need to consider a case where a deployer thinks they have a better way to wire initializers.  In such a case, they should be able to disable the compiled in initializer plugin and provide their own via webhook.  I'd probably say that our initializer plugin will always zero the list and stomp it to whatever we want, but a user *could* disable it and provide their own if they wished.

Reply:
Sent https://github.com/kubernetes/kubernetes/pull/55976.